digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -436,7 +436,7 @@ private static String toLogStreamName(final ObjectNode previousStatus)
 
     private boolean isRunningLongerThanTTL(final ObjectNode previousStatus)
     {
-        long creationTimestamp = previousStatus.get("pod_creation_timestamp").asLong();
+        long creationTimestamp = previousStatus.get("task_creation_timestamp").asLong();
```

Comment:
During migration (== restart of Digdag server application with new code), application could handle old status that contains `pod_creation_timestamp`?  If so, I think this PR needs to take care of old key name?